torchvision/ops/poly_loss.py

```diff
@@ -0,0 +1,66 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+from ..utils import _log_api_usage_once
+
+
+def poly_loss(
+    x: Tensor,
+    target: Tensor,
+    eps: float = 2.0,
+    weight: Optional[Tensor] = None,
+    ignore_index: int = -100,
+    reduction: str = "mean",
```

Comment:
I also had a thought of making this into an Enum say LossReductionMode.

Reply:
Though I do prefer enums to strings, keeping this a string will ensure strong alignment with Core and will assist on a possible upstreaming on the future.

Reply:
I'm suggesting to support both strings and LossReductionMode. But that's ok,

Reply:
I hesitated as well, but went for the same option as core